src/views/Pipeline.vue

```diff
@@ -608,6 +608,9 @@ export default defineComponent({
       this.freqType = appFreqType ? appFreqType[1] : "default"
 
       await this.store.dispatch('job/updateCurrentJob', { job });
+      if (job.runtimeData?.configId) {
```

Comment:
We can add a check here that we only need to fetch this logs for history jobs only.